config/sampleCfg_UL16.cfg

```diff
@@ -1,145 +1,152 @@
+[evaluation]
+## base = /eos/home-t/tokramer/hhbbtautau/evaluation_test/2016/
+base = /data_CMS/cms/alves/HHresonant_SKIMS/EVALS_UL16_COPIED/
```

Comment:
the `sampleCfg` should probably be set by default for the MIB samples for 2016(APV) and UHH samples for 2017